player/low-latency-streaming/index.html

```diff
@@ -87,3 +87,12 @@
         </div>
     </div>
 </div>
+
+<div class="demo-row">
+    <div class="col-md-6 chart">
+        <div class="chart" id="buffer-chart"></div>
+    </div>
+    <div class="col-md-6 chart">
+        <div class="chart" id="bitrate-chart"></div>
+    </div>
+</div>
```

Comment:
```suggestion
<div class="row demo-row">
    <div class="col-lg-6 chart">
        <div class="chart" id="buffer-chart"></div>
    </div>
    <div class="col-lg-6 chart">
        <div class="chart" id="bitrate-chart"></div>
    </div>
</div>
```
Please add the "row" class for bootstrap and i would prefer the charts be lg-6 instead of md-6 so they will do the layout change at the same time as the `latency-status` box